src/hooks/useStakeWeiInVault.spec.tsx

```diff
@@ -92,5 +92,39 @@ describe('hooks/useStakeWeiInVault', () => {
     });
   });
 
-  // TODO: add tests for stakeWeiInVaiVault and stakeWeiInVrtVault
+  it('calls stakeWeiInVrtVault with correct parameters when calling stake without a poolIndex and stakedTokenId is equal to "vrt"', async () => {
+    const TestComponent: React.FC = () => {
+      const { stake } = useStakeWeiInVault({
+        stakedTokenId: TOKENS.vrt.id as TokenId,
+      });
+
+      return (
```

Comment:
What is the advantage of rendering the button and clicking it over rendering a null component and calling the hook in the render method of the test?